tool/src/test/java/org/wildfly/security/tool/MaskCommandTest.java

```diff
@@ -126,12 +126,13 @@ public void testIterationAsStringValue() {
 
         String[] args = { "--secret", secret, "--salt", salt, "--iteration", "abcd" };
 
+
```

Comment:
nit: extra line

Reply:
yes just a minor @nidhiazad, there is an extra line here. But LGTM, I will merge this one. Thank you @nidhiazad !